evm/evm-processor/src/ds-rpc/filter.ts

```diff
@@ -140,6 +145,16 @@ export function filterBlock(block: Block, dataRequest: DataRequest): void {
             if (rel.transaction) {
                 include.addTransaction(log.transaction)
             }
+            if (rel.transactionLogs && log.transaction) {
+                for (let sibling of log.transaction.logs) {
+                    include.addLog(sibling)
+                }
+            }
+            if (rel.transactionTraces && log.transaction) {
+                for (let trace of log.transaction.traces) {
```

Comment:
Same as before, except this time presence of logs should be enforced [here](https://github.com/subsquid/squid-sdk/blob/16e7880a1a8428d173c6b50a3b4c9a653e47e93c/evm/evm-processor/src/ds-rpc/request.ts#L15) as well.